language-mapping.go

```diff
@@ -0,0 +1,50 @@
+package ctags
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+var SupportedLanguages = [...]string{"Basic", "C", "C#", "C++", "Clojure", "Cobol", "CSS", "CUDA", "D", "Elixir", "elm", "Erlang", "Go", "GraphQL", "Groovy", "haskell", "Java", "JavaScript", "Jsonnet", "kotlin", "Lisp", "Lua", "MatLab", "ObjectiveC", "OCaml", "Pascal", "Perl", "Perl6", "PHP", "Powershell", "Protobuf", "Python", "R", "Ruby", "Rust", "scala", "Scheme", "Sh", "swift", "SystemVerilog", "Tcl" /* "Thrift", */, "typescript", "tsx", "Verilog", "VHDL", "Vim"}
+
+func ListLanguageMappings(ctx context.Context, bin string) (map[string][]string, error) {
+	if bin == "" {
+		bin = "universal-ctags"
+	}
+
+	args := make([]string, 0, len(ctagsArgs)+2)
+	args = append(args, ctagsArgs...)
+	args = append(args, "--languages="+strings.Join(SupportedLanguages[:], ","))
```

Comment:
While you are here, this code is so close to allowing us implementing https://github.com/sourcegraph/go-ctags/issues/6